examples/prom/elliptic_eigenproblem_global_rom.cpp

```diff
@@ -458,6 +461,32 @@ int main(int argc, char *argv[])
         lobpcg->SetMassMatrix(*M);
         lobpcg->SetOperator(*A);
 
+        if (prescribe_init && (fom || (offline && id > 0)))
+        {
+            HypreParVector** snapshot_vecs = new HypreParVector*[nev];
+            for (int i = 0; i < nev; i++)
+            {
+                std::string snapshot_filename = baseName + "ref_snapshot_" + std::to_string(i);
+                std::ifstream snapshot_infile(snapshot_filename + ".0");
+                if (!snapshot_infile.good())
+                {
+                    if (myid == 0) std::cout << "Failed to load " << snapshot_filename << std::endl;
+                    break;
+                }
+                else
+                {
+                    snapshot_vecs[i] = new HypreParVector();
+                    snapshot_vecs[i]->Read(MPI_COMM_WORLD, snapshot_filename.c_str());
+                    if (myid == 0) std::cout << "Loaded " << snapshot_filename << std::endl;
+                    if (i == nev - 1)
```

Comment:
It would be a little clearer to put this after the loop, rather than checking for the end of the loop.

Reply:
Done.